conf/adcm/config.yaml

```diff
@@ -79,9 +79,8 @@
           display_name: "Max file size"
           description: |
             Specifies the allowed size (in Mb) the log file can reach before it is archived
```

Comment:
I think we need to remove "(in Mb)" from the description

Reply:
Done, also fix arguments in conf_file_args